src/aura/STG_CMP_Affl/STG_CMP_AfflHelper.js

```diff
@@ -1,101 +1,108 @@
 ({
```

Comment:
Hi @lifewithryan, I did some eyeball comparison but just want to make sure the only change you made in this class is format change.

Reply:
Correct, purely cosmetic at this point. Originally there were changes but after moving all of the picklist building out of the backend and into the component itself, I no longer needed it where it was, so everything left here is my cosmetic stuff.